AggregateMetrics/AggregateMetrics/AzureWebApp/FlexiblePerformanceCounterGauge.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using Microsoft.ApplicationInsights.DataContracts;
+using System.Net.Http;
+using System.Runtime.Caching;
+using System.Text.RegularExpressions;
```

Comment:
order using statements. If you'll build in Release mode - StyleCop will complain